src/drive/web/modules/filelist/FilenameInput.jsx

```diff
@@ -67,7 +67,7 @@ class FilenameInput extends Component {
     const { value } = this.state
     const { file } = this.props
     this.setState({ working: true, error: false })
-    if (!this.fileNameOnMount) this.save()
+    if (!this.fileNameOnMount) return this.save()
```

Comment:
I did some tests but couldn't file an issue with this with non directory files. Do you think it can create one?